pkg/secrets/secrets.go

```diff
@@ -237,7 +237,7 @@ func addSecretsFromMountsFile(filePath, mountLabel, containerWorkingDir, mountPr
 	return mounts, nil
 }
 
-// addFIPSModeSecret creates /run/secrets/system-fips in the container
+// addFIPSsModeSecret creates /run/secrets/system-fips in the container
```

Comment:
This looks like a function name, and the original looks correct

Reply:
@mheon I made the comment follow the function name. I didn't try to break anything. :)

Reply:
@jwhonce Could I convince you to rename the function to match the original comment, then :smile: 

It looks like a typo in the actual code the way it is now